thorlcr/activities/rollup/throllup.cpp

```diff
@@ -43,7 +43,7 @@ CActivityBase *createDedupRollupActivityMaster(CMasterGraphElement *container)
     {
         if (TAKdedup == container->getKind())
         {
-            if (((IHThorDedupArg *)container->queryHelper())->compareAll())
+            if (((IHThorDedupArg *)container->queryHelper())->getFlags() & DFcompareall)
```

Comment:
You can retain an inline function

    inline bool compareAll()  { return (getFlags() & DFcompareall) != 0; }

to reduce the number of other places that need changing.